doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -434,10 +434,19 @@ Bluetooth Fast Pair samples
 
 * :ref:`fast_pair_locator_tag` sample:
 
-  * Added possibility to build and run the sample without the motion detector support (with the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_MOTION_DETECTOR` Kconfig option disabled).
+  * Added:
+
+    * The integration of the :ref:`bt_fast_pair_adv_manager_readme` helper module (:kconfig:option:`CONFIG_BT_FAST_PAIR_ADV_MANAGER`), which replaces the application module for managing Fast Pair advertising.
+      The samples uses the new module with the locator tag extension (:kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_LOCATOR_TAG`) that automates common advertising scenarios for this use case.
```

Comment:
```suggestion
    * The integration of the :ref:`bt_fast_pair_adv_manager_readme` helper module (:kconfig:option:`CONFIG_BT_FAST_PAIR_ADV_MANAGER`) that replaces the application module for managing Fast Pair advertising.
      The samples use the new module with the locator tag extension (:kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_LOCATOR_TAG`) that automates common advertising scenarios for this use case.
```

Reply:
oh sorry, should be singular sample (The sample uses)